src/zhinst/toolkit/driver/devices/base.py

```diff
@@ -120,12 +120,13 @@ def _version_string_to_tuple(version: str) -> t.Tuple[int, int, int]:
         Returns:
             Version as a tuple of ints
         """
-        result = [0] * 3
-        for i, value in enumerate(version.split(".")):
+        version_split = version.split(".")
```

Comment:
comment:

If we'd know the version is always at least 3, we could do:

```
major, minor, micro, *_ = version.split(".")
return major, minor, micro
```

Reply:
how about this: 
return tuple(int(v) if v.isdigit() else 0 for v in version.split(".")[:3])